invenio_base/translations/fr/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,26 @@
+# Translations template for invenio-base.
+# Copyright (C) 2016 CERN
+# This file is distributed under the same license as the invenio-base
+# project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2016.
+# 
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: invenio-base 1.0.0a15.dev20160823\n"
+"Report-Msgid-Bugs-To: info@inveniosoftware.org\n"
+"POT-Creation-Date: 2016-08-24 15:03+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Alizee Pace <alizee.pace@gmail.com>, 2016\n"
+"Language-Team: French (https://www.transifex.com/inveniosoftware/teams/23537/fr/)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.3.4\n"
+"Language: fr\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: invenio_base/cookiecutter-invenio-base/{{ cookiecutter.site_name }}/{{
+#: cookiecutter.package_name }}/config.py:28
+msgid "{{ cookiecutter.site_name }}"
```

Comment:
It's suprising we'd ask anyone to translate `{{ cookiecutter.site_name }}`...  A misplaced `_(...)` around string?